hooks/post_gen_project.py

```diff
@@ -47,6 +47,18 @@
     "python-dotenv",
 ]
 
+# Use the selected documentation package specified in the config,
+# or none if none selected
+docs_path = Path("docs")
+# {% if cookiecutter.docs != "none" %}
+packages += ["{{ cookiecutter.docs }}"]
+pip_only_packages += ["{{ cookiecutter.docs }}"]
+docs_subpath = docs_path / "{{ cookiecutter.docs }}"
+for obj in docs_subpath.iterdir():
+    shutil.move(str(obj), str(docs_path))
+# {% endif %}
+shutil.rmtree(docs_path / "mkdocs")
```

Comment:
Maybe should be a for loop that removes all the other options for `cookiecutter.docs` that were not selected if they exist so we have a little future proofing.